tests/config/pytest_software.yml

```diff
@@ -281,9 +301,15 @@ ivar_variants:
   - software/ivar/variants/**
   - tests/software/ivar/variants/**
 
+<<<<<<< HEAD
+kb_ref:
+  - software/kb/ref/**
+  - tests/software/kb/ref/**
+=======
 kallisto_index:
   - software/kallisto/index/**
   - tests/software/kallisto/index/**
+>>>>>>> upstream/master
```

Comment:
Oops. Looks like you have merge conflicts here @FloWuenne . I suspect it may be easier to just create the module from scratch in a separate PR with the full name. You should be able to copy and paste most of the main content you wrote for the module.